src/docs/sphinx/quickstart.rst

```diff
@@ -177,40 +177,72 @@ container. Instructions for this process are located :ref:`here <docker-label>`.
 Building Serac
 --------------
 
-Serac uses a CMake build system that wraps its configure step with a script
-called ``config-build.py``.  This script creates a build directory and
-runs the necessary CMake command for you. You just need to point the script
-at the generated or a provided host-config. This can be accomplished with
-one of the following commands:
+Serac is a CMake project. Configure with at least the ``-C``
+flag below, to ensure CMake can locate the dependencies installed by spack:
 
 .. code-block:: bash
 
-   # If you built Serac's dependencies yourself either via Spack or by hand
-   $ python3 ./config-build.py -hc <config_dependent_name>.cmake
+  $ cmake . -B build \
+            -C path/to/serac/<machine>-<type>-<compiler>.cmake \
+            -DCMAKE_BUILD_TYPE=Release
 
-   # If you are on an LC machine and want to use our public pre-built dependencies
-   $ python3 ./config-build.py -hc host-configs/<machine name>-<SYS_TYPE>-<compiler>.cmake
+LLNL developers working on one of the LC machines can instead use one of 
+the provided CMake cache files in serac's ``host-configs`` directory:
 
-   # If you'd like to configure specific build options, e.g., a release build
-   $ python3 ./config-build.py -hc /path/to/host-config.cmake -DCMAKE_BUILD_TYPE=Release <more CMake build options...>
+``-C path/to/serac/host-configs/<machine>-<type>-<compiler>.cmake``
 
-If you built the dependencies using Spack/uberenv, the host-config file is output at the
-project root. To use the pre-built dependencies on LC, you must be in the appropriate
-LC group. Contact `Jamie Bramwell <bramwell1@llnl.gov>`_ for access.
+----------
+ 
+Some other frequently-useful configure flags
 
-Some build options frequently used by Serac include:
+``-B <build-directory>`` (recommended) 
```

Comment:
This is not just recommended, it is required. We do not allow in-source builds, which leaving off the `-B` option turns into.

Reply:
Sure. I meant that the common pattern of
`mkdir build && cd build && cmake .. <args>`
doesn't require the -B flag, but I'll clarify.